src/integrationTest/java/org/opensearch/security/api/InternalUsersRestApiIntegrationTest.java

```diff
@@ -59,8 +61,10 @@ public class InternalUsersRestApiIntegrationTest extends AbstractConfigEntityApi
 
     private final static String SOME_ROLE = "some-role";
 
-    private final PasswordHasher passwordHasher = new BCryptPasswordHasher();
-
+    private final PasswordHasher passwordHasher = PasswordHasherFactory.createPasswordHasher(Settings
+            .builder()
+            .put(ConfigConstants.SECURITY_PASSWORD_HASHING_ALGORITHM, ConfigConstants.BCRYPT)
```

Comment:
nit: should we just say ```PASSWORD_HASHING_ALGORITHM"?

Reply:
I am happy to change it though I see that most of other constants in that file have the "SECURITY_" prefix

Reply:
Ah ok. then i think what you have is good.